packages/framework/data-object-base/tsconfig.es5.json

```diff
@@ -2,16 +2,11 @@
     "extends": "./tsconfig.json",
     "compilerOptions": {
         "outDir": "./es5",
-        "target": "ES5",
```

Comment:
same here

Reply:
yeah, the build would not pass otherwise